src/main/java/service/TicketService.java

```diff
@@ -0,0 +1,81 @@
+package service;
+
+import DAO.TicketDAO;
+import model.*;
+
+import model.enums.TicketType;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+import view.Printable;
+
+import java.util.List;
+import java.util.UUID;
+
+
+@Component
+@Transactional
```

Comment:
when you use this annotation with class level, then all methods will be performed under transactions. In some cases, for example when we have READ-only opeations from db, we do not need to do it with transaction. Better use this annotation with explicitly specific methods where transaction is required.

Reply:
Fixed. I add annotation @Transactional only to methods, that update / delete information from DB.